libsolidity/formal/CHC.cpp

```diff
@@ -900,13 +898,11 @@ void CHC::defineInterfacesAndSummaries(SourceUnit const& _source)
 						!base->isInterface()
 					)
 					{
-						auto state1 = stateVariablesAtIndex(1, *base);
-						auto state2 = stateVariablesAtIndex(2, *base);
+						auto state1 = stateVariablesAtIndex(1, *contract);
+						auto state2 = stateVariablesAtIndex(2, *contract);
 
-						auto const* iface = m_nondetInterfaces.at(base);
-						auto state0 = stateVariablesAtIndex(0, *base);
-						auto nondetPre = (*iface)(state0 + state1);
-						auto nondetPost = (*iface)(state0 + state2);
+						auto nondetPre = iface(state0 + state1);
+						auto nondetPost = iface(state0 + state2);
```

Comment:
Code is the same, in this diff block, the first 2 lines were just moved outside the loop.